### ISSUE TITLE
Testing for errors shouldn't depend on the UI message

### ISSUE THREAD

**Issue by czerwinskilukasz1** [xFAANG/askql#252]
Tests like
https://github.com/xFAANG/askql/blob/54346cdb33bc160600eb336592c92dbc0698f250/src/askscript/__tests__/90-misc/scope-02.test.ts
depend on a string content of a *message* displayed to a user instead of on *error type*.

**Comment by mhagmajer** [comment]
Good point! I added https://github.com/xFAANG/askql/issues/242 to figure out different types of errors
